sources/libcpp83gts_callback_and_action/memory_config.cpp

```diff
@@ -0,0 +1,194 @@
+#include "memory_config.h"
+
+const char* memory_config::str_on_  = "ON";
```

Comment:
Maybe you don't need to separate initialization from declaration. Plz write them in header.